.travis.yml

```diff
@@ -1,11 +1,9 @@
-sudo: false
 language: node_js
 node_js:
-  - "0.12"
+  - "10"
 addons:
   apt:
     packages:
     - xsltproc
 before_script:
-  - npm install -g grunt-cli
```

Comment:
Is `grunt-cli` no longer needed?

Reply:
We have it in `devDependencies` which is a recommended practice as you don't depend on global packages that may change any moment.

Reply:
Ahh ok. Something I've learned today ;-)